### PR TITLE
chore(deps) Bump 202208 dependabot upgrades at once

### PR DESCRIPTION
**Pull request by Soulou**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -4,20 +4,20 @@ go 1.17
...
 	github.com/Scalingo/go-plugins-helpers v1.3.0
-	github.com/Scalingo/go-utils/etcd v1.1.0
+	github.com/Scalingo/go-utils/etcd v1.1.1
 	github.com/Scalingo/go-utils/graceful v1.1.0
-	github.com/Scalingo/go-utils/logger v1.1.0
+	github.com/Scalingo/go-utils/logger v1.1.1
 	github.com/bits-and-blooms/bitset v1.2.2
-	github.com/docker/docker v20.10.16+incompatible
+	github.com/docker/docker v20.10.17+incompatible
 	github.com/docker/libnetwork v0.8.0-dev.2.0.20171213192018-26531e56a76d
```
Comment (EtienneM) [inline go.mod:14]: Did you try to check if there is an update for this dependency?

**Review by EtienneM — APPROVED** [review]

File: go.mod (hunk 1 above)
Comment (Soulou) [reply go.mod:14]: Another update? I don't get it @EtienneM

Comment (EtienneM) [reply go.mod:14]: The dependency `github.com/docker/libnetwork` has a pseudo-version number (`v0.8.0-dev.2.0.20171213192018-26531e56a76d`). Dependabot does not handle that kind of dependency. My question is whether there is an update available for this dependency? The current version dates back to 2017, that's old.

Comment (Soulou) [reply go.mod:14]: Oh ok, I didn't know that, I'm checking

Comment (Soulou) [reply go.mod:14]: I'll do another PR to remove this dependency actually, we don't really use it

**Pull request merged**